### ISSUE TITLE
The venue UI should match the wireframes

### ISSUE THREAD

**Issue by ctrembath** [club-soda/club-soda-guide#230]
I've been reviewing the wireframes from Harry and have noticed that the Venue UI hasn't been implemented or discussed in any existing issues.

[Venues-compressed.pdf](https://github.com/club-soda/club-soda-guide/files/2676112/Venues-compressed.pdf)

We had assumed that the UI would be implemented with the build-out of the venue listing journey.

Can you update me one what the plan is with this?

**Comment by iteles** [comment]
@ctrembath 👋 Hi, welcome back!
The venue setup UI was the last thing Harry put together and was something we discussed at the time to use as a guide for thinking through the fields and possible starts to user flows - as you may remember, it wasn't part of the original scope (we just had venue imports in the initial scope we discussed). This was something we emphasised quite a lot through the design process and as you know is one of the dangers of having designs created before development has been able to sink their teeth into the project at all.

At the point that it _did_ become a priority (though we didn't discuss how this would affect the rest of the scope, which we have built out anyway), any decisions that we discussed on calls were reflected in changes to acceptance criteria wherever possible. It's all quite spread out and not always clear so I'm very happy to provide an update :blush:

Probably the change that feels the biggest that we discussed was the deprioritisation of creating a full user management system from the get-go. The database is append-only so data is only ever added to, not lost. Functionality took precedence and we landed on #158 and tonight we will also be implementing #228. A full user management system will be implemented in the new year.

With regards to the wireframes, there were sparse changes, all discussed as we went through demos and issues:
+ Email and the initial form are input together to ensure we capture key data
  + Password will come in as part of #53 and phone number was considered non-essential for this initial form which as discussed is to be an non-onerous as possible
+ We agreed (I believe still with you) that the lookup functionality for the postcode was a lower priority #100 and the request followed for the first line of the address following a demo #147 
+ The holding page for the venue is much the same (the flow also has not changed) with a few notes:
  + Opening hours were removed for now #119 
  + Map functionality is to be implemented as it was deemed a lower priority #197 #221 
  + Social media links were added to the scope as requested #147 
  + Nearby hotspots is one to discuss as it was out of original scope but is either way dependent on #6 
+ The process for adding drinks was considerably streamlined from the designs and there was a lot of conversation on this, with #92 being the focus https://github.com/club-soda/club-soda-guide/issues/60#issuecomment-430234899   #105 #107
+ We spent quite a lot of time on calls discussing claiming of venues but in the end the complexity justified a concierge MVP approach for now through user testing: https://github.com/club-soda/club-soda-guide/issues/30

There are still styling tweaks to be finalised which we will do as a final sweep of the whole app.

I hope this answers the question! Let me know if you have any specific questions on any part ✨

**Comment by ctrembath** [comment]
Thanks for explaining!
